composer.json

```diff
@@ -31,11 +31,7 @@
     },
     "autoload": {
         "psr-4": {
-            "Yiisoft\\Db\\Mysql\\": "src"
-        }
-    },
-    "autoload-dev": {
```

Comment:
Why did you delete this?

Reply:
Because I am going to run the tests from each implementation in db, it currently works the other way around, and the idea is to move the tests to each implementation you can read the issue yiisoft/db#108.